Formula/swiftdraw.rb

```diff
@@ -0,0 +1,27 @@
+class Swiftdraw < Formula
+  desc "Convert SVG into PDF, PNG, JPEG or SF Symbol"
+  homepage "https://github.com/swhitty/SwiftDraw"
+  url "https://github.com/swhitty/SwiftDraw/archive/0.12.0.tar.gz"
+  sha256 "48397fa5761abc542f54739ee5bc9657d3646e8dced1043460ed18dd77f3db1e"
+  license "Zlib"
+  head "https://github.com/swhitty/SwiftDraw.git", branch: "main"
+
+  depends_on xcode: ["12.5", :build]
+  uses_from_macos "swift"
+
+  def install
+    system "swift", "build", "--disable-sandbox", "--configuration", "release"
+    bin.install ".build/release/swiftdraw"
+  end
+
+  test do
+    (testpath/"fish.svg").write <<~EOS
+      <?xml version="1.0" encoding="utf-8"?>
+      <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="160" height="160">
+        <path d="m 80 20 a 50 50 0 1 0 50 50 h -50 z" fill="pink" stroke="black" stroke-width="2" transform="rotate(45, 80, 80)"/>
+      </svg>
+    EOS
+    system "#{bin}/swiftdraw", "#{testpath}/fish.svg", "--format", "sfsymbol"
+    assert_path_exists "#{testpath}/fish-symbol.svg"
```

Comment:
```suggestion
    system bin/"swiftdraw", testpath/"fish.svg", "--format", "sfsymbol"
    assert_path_exists testpath/"fish-symbol.svg"
```
We don't need interpolation here.